External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -41,6 +42,10 @@ public class ASGenerator
         static private MemberModel contextMember;
         static private bool firstVar;
 
+        static List<ICompletionListItem> known;
+
+        public static List<ICompletionListItem> KnownList => known;

```

Comment:
Well, such are things when it comes to new software. I even know of companies that mainly work with .NET and their main VS version is 2010.